variables.tf

```diff
@@ -1,5 +1,14 @@
 data "aws_availability_zones" "available" {}
 
+data "aws_acm_certificate" "default" {
+  domain      = "${var.site_name}"
+  most_recent = true
+}
+
+variable "site_name" {
```

Comment:
I'm not sure about keeping this since it only adds a description... what do you think @GustavoTango?